models/mock_model.py

```diff
@@ -18,22 +19,52 @@ def __init__(self) -> None:
         self.lv_battery = 88
         self.current = 7.6
         self.is_burning = 0
-        self.is_charging = 0
         self.is_debug = 0
         self.bms_faults = 0
         self.mpu_faults = 0
+        self.max_cell_voltage = 3.5
+        self.max_cell_voltage_chip_number = 1
+        self.max_cell_voltage_cell_number = 10
+        self.max_cell_temp = 30
+        self.max_cell_temp_chip_number = 5
+        self.max_cell_temp_cell_number = 8
+        self.min_cell_voltage = 3.2
+        self.min_cell_voltage_chip_number = 2
+        self.min_cell_voltage_cell_number = 3
+        self.min_cell_temp = 25
+        self.min_cell_temp_chip_number = 3
+        self.min_cell_temp_cell_number = 4
+        self.average_cell_voltage = 3.3
+        self.average_cell_temp = 27
+        self.pack_voltage = 3.4
+        self.bms_state = 0
+        self.pack_current = 4.6
+        self.dcl = 280
+        self.ccl = 300
+        self.gforce_x = .5
+        self.gforce_y = -1
+        self.gforce_z = .5
+        self.sd_card_status = 4
+        self.segement1_temp = 30
+        self.segement2_temp = 50
+        self.segement3_temp = 35
+        self.segement4_temp = 15
+        self.motor_power = 100
+        self.fan_power = 100
+        self.torque_power = 100
+        self.regen_power = 1
+        self.mode_index = 0
+        self.state_of_charge_delta = -1
+        self.inverter_temp = 30
         self.current_data = [self.mph, self.status, self.dir, self.pack_temp,
```

Comment:
get rid of this and just make the get functions return teh actual values

it's a mock model, it's just supposed to output the right stuff the logic doesnt need to be similar at all

Reply:
lets have new ppl do this